repository-data/application/src/main/resources/hcm-config/namespaces/hee/tabPanel.yaml

```diff
@@ -62,13 +69,25 @@ definitions:
               jcr:primaryType: frontend:pluginconfig
           /tabBody:
             jcr:primaryType: frontend:plugin
-            caption: Tab body
+            caption: Tab content
```

Comment:
Spec seems to indicate that the caption should be `Content`. I personally think that it should be named `Tab content` (like you did), but the spec needs to be updated to reflect it in that case.